### DISCUSSION TITLE
How do I zoom my browser using SeleniumBase?

### DISCUSSION THREAD

**Discussion by elnatan2604** [seleniumbase/SeleniumBase#2855] [Q&A] [answered]
Im trying to zoom my browser and I used this line:
self.execute_script("document.body.style.zoom = '0.75'")

but it zoomed the web elements and not the browser.
Is there any way using seleniumbase to do this?

**Comment by mdmintz** [comment, marked as answer]
It would be the same as regular Selenium.
Stack Overflow: https://stackoverflow.com/a/32135331/7058266
